api_core/google/api_core/bidi.py

```diff
@@ -200,7 +203,7 @@ def open(self):
         request_generator = _RequestQueueGenerator(
             self._request_queue, initial_request=self._initial_request
         )
-        call = self._start_rpc(iter(request_generator))
+        call = self._start_rpc(iter(request_generator), metadata=self._rpc_metadata)
```

Comment:
We need unit test coverage here.

Reply:
Actually, we already have [failing tests here](https://source.cloud.google.com/results/invocations/fa903b7b-49d9-41d7-99a3-3a71942b275b/targets/cloud-devrel%2Fclient-libraries%2Fgoogle-cloud-python%2Fpresubmit%2Fapi_core/log).